drivers/usbhid-ups.c

```diff
@@ -1493,6 +1504,12 @@ static bool_t hid_ups_walk(walkmode_t mode)
 			hd = NULL;
 			return FALSE;
 
+		case LIBUSB_ERROR_IO:        /* I/O error */
+			/* Uh oh, got to reconnect, with a special suggestion! */
+			interrupt_pipe_EIO_count++;
```

Comment:
Wondering if the count should go down/zero if there were successful reads later?.. So far this is mostly informative (suggest pollonly if there were any interrupt errors)...